MIG.HomeAutomation/ZWave.cs

```diff
@@ -253,7 +266,7 @@ public bool IsConnected
 
         public object InterfaceControl(MigInterfaceCommand request)
         {
-            ResponseText returnValue = new ResponseText("OK");
+            object returnValue = "OK";
             bool raiseEvent = false;
```

Comment:
Please keep back the use of ResponseText. Also code in the zwave javascript needs to be reverted, eg: https://github.com/genielabs/HomeGenie/pull/260/files#diff-a70ab7bc7299900b81931017a2304bd3L546